.travis.yml

```diff
@@ -24,7 +24,11 @@ before_script:
 
   # Install composer dependencies
   - composer validate
-  - composer require --no-update silverstripe/recipe-core:4.6.x-dev silverstripe/versioned:1.6.x-dev
+  - composer require --no-update silverstripe/recipe-core:$RECIPE_VERSION
+  # silverstripe/versioned must be 1.x-dev (or 1.n.x-dev), not ^1
+  - composer require --no-update silverstripe/versioned:1.x-dev
```

Comment:
It used to be `1.6.x-dev`, not `1.x-dev`

Reply:
Updated to `1.6.x-dev`